<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-HCMJITA" ignored="5" name="Default test" tests="1" failures="1" timestamp="2021-12-12T22:37:09 IST" time="18.955" errors="0">
  <testcase name="loginlinktest" time="0.058" classname="com.qa.opencart.test.Logintest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.opencart.test.Logintest.loginlinktest(Logintest.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginlinktest -->
  <testcase name="loginappTest" time="0.0" classname="com.qa.opencart.test.Logintest">
    <ignored/>
  </testcase> <!-- loginappTest -->
  <testcase name="forgottenpswdlintest" time="0.0" classname="com.qa.opencart.test.Logintest">
    <ignored/>
  </testcase> <!-- forgottenpswdlintest -->
  <testcase name="clickloginbn" time="0.0" classname="com.qa.opencart.test.Logintest">
    <ignored/>
  </testcase> <!-- clickloginbn -->
  <testcase name="loginopencart" time="0.0" classname="com.qa.opencart.test.Logintest">
    <ignored/>
  </testcase> <!-- loginopencart -->
  <testcase name="loginpagetitleTest" time="0.0" classname="com.qa.opencart.test.Logintest">
    <ignored/>
  </testcase> <!-- loginpagetitleTest -->
</testsuite> <!-- Default test -->
